apps/web/playwright/teams.e2e.ts

```diff
@@ -62,4 +64,43 @@ test.describe("Teams", () => {
       // await expect(page.locator('[data-testid="empty-screen"]')).toBeVisible();
     });
   });
+  test("Can create a booking", async ({ page, users }) => {
+    const user = await users.create({ username: "pro-user" }, { hasTeam: true });
+    const teamMate1 = await users.create();
+    const teamMate2 = await users.create();
+
+    // TODO: Create a fixture and follow DRY
+    const { team } = await prisma.membership.findFirstOrThrow({
+      where: { userId: user.id },
+      include: { team: true },
+    });
+
+    // Add teamMate1 to the team
+    await prisma.membership.create({
+      data: {
+        teamId: team.id,
+        userId: teamMate1.id,
+        role: MembershipRole.MEMBER,
+        accepted: true,
+      },
+    });
+
+    // Add teamMate2 to the team
+    await prisma.membership.create({
+      data: {
+        teamId: team.id,
+        userId: teamMate2.id,
+        role: MembershipRole.MEMBER,
+        accepted: true,
+      },
+    });
```

Comment:
Whats the teamMate2 role in this test? Should we edit event type to be collective event so the success booking have 2 team attendees and 1 guest?

Reply:
I think that is a valid point. We should consider a valid schedulingType.